lib/index.js

```diff
@@ -104,16 +104,6 @@ function getComputerFingerprint(callback, append) {
  */
 function PubSubClient(opts) {
 	opts = opts || {};
-	opts.debug && (debug = function () {
```

Comment:
@jareddlc fine to strip the colors.  Let's not strip the prefixing.

Reply:
prefix is intact: https://github.com/jareddlc/appc-pubsub/blob/5b9c62b993e4856855de305091100dc8b0049dfe/lib/index.js#L12

Reply:
@jareddlc so it is 🙄  Tx.